docs/tracker/tracker_bytetrack_detections.py

```diff
@@ -0,0 +1,57 @@
+from tqdm.notebook import tqdm
+from ultralytics import YOLO
```

Comment:
@SkalskiP @dbroboflow `import ultralytics` package might create an issue for license. Please verify it once.

Reply:
These are just for test scripts. When we are merging it, we will delete it. Also I added it to the docs. We don't need to install ultralytics package.